gui/webdisplay/inc/ROOT/TWebWindow.hxx

```diff
@@ -143,6 +144,9 @@ public:
    /// Returns current number of active clients connections
    unsigned NumConnections() const { return fConn.size(); }
 
+   // Return connection id for given connection
+   unsigned GetConnectionId(unsigned num = 0) const;
```

Comment:
Why does `num` need a default argument?

Reply:
Majority of user applications will deal with only connection.
Means `NumConnections()` will return 1 and `GetConnectionId()` will return the only connection it has.